pkg/compiler/codegen.go

```diff
@@ -1325,7 +1338,19 @@ func (c *codegen) convertMap(lit *ast.CompositeLit) {
 	}
 }
 
-func (c *codegen) convertStruct(lit *ast.CompositeLit) {
+func (c *codegen) getStruct(typ types.Type) (*types.Struct, bool) {
+	switch t := typ.Underlying().(type) {
+	case *types.Struct:
+		return t, true
+	case *types.Pointer:
+		strct, ok := t.Elem().Underlying().(*types.Struct)
+		return strct, ok
```

Comment:
Just
```
return t.Elem().Underlying().(*types.Struct)
```
?

Reply:
No, this way it will assume single value return and it will not compile.